README.md

```diff
@@ -36,6 +38,41 @@ docker/join.sh mbari_lrauv
 . /home/colcon_ws/install/setup.bash
 ```
 
+
+### Empty World from Source
+
+~ 90 minutes from scratch
+
+~ 70 seconds with only workspace changes
+
+```
+cd docker
+docker build -t mbari_lrauv -f empty_world/Dockerfile ..
+rocker --nvidia --x11 --user mbari_lrauv bash
+ign launch lrauv_world.ign
```

Comment:
The launch file was removed in #102 , we should use plain `ign gazebo` commands now